docs/other/model-in-memory.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: model_in_memory
+title: Creating AMF model in memory
+---
+:::info
+The functionality expressed in this documentation requires familiarity with the
+[amf model](https://github.com/aml-org/amf/blob/develop/documentation/model.md).
+:::
+
+AMF is most commonly used to parse an already existing API,
+but it can also be used to create an API in a fluent and descriptive way.
+
+Some use cases for creating a WebApi model in memory include:
+
+- To edit an already parsed API with AMF
+- When using AMF to power a visual API designer
+- To create programmatic API tests
+
+
+### Creating a WebApi with AMF programmatically
```

Comment:
## Create a Web API model with AMF

I'm not sure how we're using the words create and model. Do you use AMF to create an API model? Or do you use AMF to create an API? Or do you use AMF to model an API? We should clarify the verb and noun use of model